### PR TITLE
[#54] Save manual entered values in a cookie

### PR DESCRIPTION
**Pull request by emilianop11**
@san983 Pls check this one

### CODE REVIEW

File: public/javascripts/controllers/widget.portal.js [hunk 1]
```diff
@@ -29,6 +29,55 @@ define([
...
+      else if (ibc === 'link_button_to_a_page')
+        $scope.widget.link_button_to_a_page = true;
+      else if (ibc === 'show_text_html')
+        $scope.widget.show_text_html = true;
+
+      $scope.widget.show_text_html_value = $cookies.get('show_text_html_value');
+      $scope.widget.link_button_to_a_page_value = $cookies.get('link_button_to_a_page_value');
+    }
+
+    function saveCookies(){
```
Comment (san983) [inline public/javascripts/controllers/widget.portal.js]: Could we just save the widget object into a cookie instead of every single setting?

Comment (emilianop11) [inline public/javascripts/controllers/widget.portal.js]: My bad!

File: tests/cookies.test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      .setValue('textarea[name="show_text_html_value"]', 'Incompatible_browser')
+
+      //refresh the page and test cookies
+      .url(browser.launchUrl+"/click2vox/?e164=1234567890&login=login&password=password&basic_auth=1")
+
+      .assert.valueContains ('input[name="button_label"]', 'My_Test_Button')
+      .assert.valueContains ('input[name="caller_id"]', 'This_is_the_test_bot')
+      .assert.valueContains ('input[name="send_digits"]', '1,2,3,1200ms,4,5,900ms,6,#')
+      .assert.valueContains ('input[name="context"]', '12context34')
+      .assert.valueContains ('textarea[name="show_text_html_value"]', 'Incompatible_browser')
```
Comment (san983) [inline tests/cookies.test.js]: This one is failing in my local test run

File: tests/generation.test.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ module.exports = {
       .url(browser.launchUrl+"/click2vox/?e164=1234567890&login=login&password=password&basic_auth=1")
       .waitForElementVisible('body', 5000)
       .assert.title('Voxbone Widget Generator v1.5.0')
-      .assert.containsText('h1', '1234567890')
+      .assert.containsText('div.portal-title', 'Create a WebRTC Call Button for 1234567890')
```
Comment (san983) [inline tests/generation.test.js]: @emilianop11 just modified this one, which was failing, I think it's a leftover from the merge from the new template 😃

**Pull request merged**